tests/test_auxtel_track_target_and_take_image.py

```diff
@@ -319,6 +320,25 @@ async def test_run_fail_check_tracking(self):
 
             self.script.atcs.stop_tracking.assert_awaited_once()
 
+    async def test_run_fail_assert_feasibility(self):
+
+        async with self.make_script(), self.setup_mocks():
+
+            self._ataos_evt_correction_enabled.hexapod = False
+
+            await self.configure_script_full()
+
+            with pytest.raises(AssertionError):
+                await self.run_script()
+
```

Comment:
Maybe add a comment why you make these assertions?
Presumably because you want to make sure nothing moves and the script fails immediately?

Reply:
Uhm, not sure I understand your question.. Here I am testing that the script will fail the way I want it to fail given the condition imposed by the test. In this case, that it raises an `AssertionError` if the corrections are disabled.

Reply:
i meant the assertions below that are not in the `with` statement.
`self.script.atcs.slew_icrs.assert_not_awaited()`

Reply:
ah.. so here I only add to the `with` statement the code that actually belongs there. Because here I want to check that `await self.run_script()` raises `AssertionError` not the rest...

Reply:
I understand the `with` statement... that's not the issue, sorry for being unclear.
What I mean is that there are 5 assertions below (4 `assert_not_awaited` and 1 `assert_awaited_once` and no explanation of why you are testing those.... and where the "once" comes from.

Reply:
Ok, I think I see what you are asking.. The following assertions are basically checking what I expect the code to do when this condition happens.. I though it was a bit self explanatory, but maybe I have that impression because I wrote it.. 

I can add a comment if you feel like it needs some clarification...